pipelines.go

```diff
@@ -121,6 +121,7 @@ type PipelineTestCases struct {
 	RecentFailures RecentFailures `json:"recent_failures"`
 }
 
+// SystemOutput contains information about test cases when it fails
```

Comment:
```suggestion
// SystemOutput contains information about test cases when it fails.
```
Following the Go coding guides and the style in this package 😏